client/demo/designsystem-demo.js

```diff
@@ -173,6 +170,24 @@ function ProfileScreen() {
                     </SpacedArray>
                 </Datastore>
             </DemoSection>
+            <DemoSection label='Verified Profile Photo'>
+            <Datastore personaPreview={{ verificationStatus: 'verified' }}>
```

Comment:
This seems to only be testing the case where we show the badge of the current logged in user for an instance that they aren't attached to.

We should test that use case, but the most important use case is the case where you are seeing the profile photo for a different user - and in that case, their persona info will be in the persona collection.

Reply:
done